kernel/multicore_bringup/src/aarch64.rs

```diff
@@ -50,9 +50,29 @@ pub fn handle_ap_cores(
     ap_data.ap_data_virt_addr.write(VirtualAddress::new_canonical(&ap_data as *const _ as usize));
     ap_data.ap_stage_two_virt_addr.write(VirtualAddress::new_canonical(ap_stage_two as usize));
 
+    // -------
+    //
+    // This code identity-maps one page of writable+executable memory and
+    // copies the machine code of `ap_entry_point` to it. The fact that the
+    // mapping is identity-mapped allows the code to enable the MMU safely,
+    // as PC will be valid before and after the switch.
+
+    // map this RWX
+    let flags = PteFlags::new().valid(true).writable(true).executable(true);
```

Comment:
This mapping obviously needs to be writable initially for us to copy stuff to it. But does it need to be writable after that initial copy, or can we remap it as just R+X? 

Generally we want to map pages using the _minimum_ required permissions, and if those pages just contain executable text content, then we definitely don't want to map them as writable.

Reply:
> can we remap it as just R+X?
yes will do